test/test_helper/common.bash

```diff
@@ -205,3 +206,34 @@ function wait_for_empty_mail_queue_in_container() {
     # shellcheck disable=SC2016
     repeat_in_container_until_success_or_timeout "${TIMEOUT}" "${CONTAINER_NAME}" bash -c '[[ $(mailq) == *"Mail queue is empty"* ]]'
 }
+
+# Common defaults appropriate for most tests, override vars in each test when necessary.
+# TODO: Check how many tests need write access. Consider using `docker create` + `docker cp` for easier cleanup.
+function init_with_defaults() {
+  # Ignore absolute dir path and file extension, only extract filename:
+  export TEST_NAME
+  TEST_NAME="$(basename "${BATS_TEST_FILENAME}" '.bats')"
+
+  export PRIVATE_CONFIG
+  PRIVATE_CONFIG="$(duplicate_config_for_container . "${TEST_NAME}")"
+  export TEST_FILES_VOLUME="${PWD}/test/test-files:/tmp/docker-mailserver-test:ro"
+  export TEST_CONFIG_VOLUME="${PRIVATE_CONFIG}:/tmp/docker-mailserver:ro"
+
+  export TEST_FQDN='mail.my-domain.com'
```

Comment:
May `mail.example.com` not a better choise?

Reply:
Oh for sure. I use `mail.example.test` myself, the cipherlist test uses such a domain with the self-signed certs for such a domain.

I left it as this though since many of the existing tests that do involve a domain name in their tests use this one I think. I'll replace it in the future as a find/replace commit for all occurrences.